### PR TITLE
Updating the Unity documentation because Microsoft haven't for the last 8 years

### PR DESCRIPTION
**Pull request by TeaDrinkingProgrammer**
Hopefully I can prevent people from wasting time on this.
https://forum.unity.com/threads/update-on-the-visual-studio-code-package.1302621/

### CODE REVIEW

File: docs/other/unity.md [hunk 1]
```diff
@@ -7,9 +7,14 @@ PageTitle: Visual Studio Code and Unity
 DateApproved: 11/4/2022
 MetaDescription: Visual Studio Code can replace MonoDevelop as the editor for Unity
 ---
+# VScode is officially unsupported
+
+![VScode is officially unsupported by Unity](https://forum.unity.com/threads/update-on-the-visual-studio-code-package.1302621/)
+For now, the only officially supported editors are ![Visual Studio](https://visualstudio.microsoft.com/vs/unity-tools/) and ![Rider](https://www.jetbrains.com/lp/dotnet-unity/).
+
 # Unity Development with VS Code
```
Comment (sille2222) [inline docs/other/unity.md:15]: hy

File: docs/other/unity.md [hunk 2]
```diff
@@ -7,9 +7,14 @@ PageTitle: Visual Studio Code and Unity
...
 
-Visual Studio Code can be a great companion to Unity for editing C# files.  All of the [C#](/docs/languages/csharp.md) features are supported and more.  In the screen below, you can see code colorization, bracket matching, IntelliSense, CodeLens and that's just the start.
+Visual Studio Code can be a great companion to Unity for editing C# files. All of the [C#](/docs/languages/csharp.md) features are supported and more.  In the screen below, you can see code colorization, bracket matching, IntelliSense, CodeLens and that's just the start.
```
Comment (cadamini) [inline docs/other/unity.md:17]:
```suggestion
VScode can be a great companion to Unity for editing C# files. All of the [C#](/docs/languages/csharp.md) features are supported and more. In the screen below, you can see code colorization, bracket matching, IntelliSense, CodeLens and that's just the start.
```

Comment (cadamini) [reply docs/other/unity.md:17]:
```suggestion
VScode can still help you edit C# files. All of the [C#](/docs/languages/csharp.md) features are supported and more. In the screen below, you can see code colorization, bracket matching, IntelliSense, CodeLens and that's just the start.
```

better?

File: docs/other/unity.md (hunk 1 above)
Comment (cadamini) [inline docs/other/unity.md:15]:
```suggestion
# Develop Unity C# files with VS Code
```

wouldn't this be a more suitable heading now?

File: docs/other/unity.md [hunk 3]
```diff
@@ -7,9 +7,14 @@ PageTitle: Visual Studio Code and Unity
 DateApproved: 11/4/2022
 MetaDescription: Visual Studio Code can replace MonoDevelop as the editor for Unity
 ---
+# VScode is officially unsupported
```
Comment (cadamini) [inline docs/other/unity.md:10]:
```suggestion
# VS Codeis officially unsupported
```

File: docs/other/unity.md (hunk 2 above)
Comment (TeaDrinkingProgrammer) [reply docs/other/unity.md:17]: That's true, thanks for the correction

**Comment by Chizaruu** [comment]
> Hopefully, I can prevent people from wasting time on this. https://forum.unity.com/threads/update-on-the-visual-studio-code-package.1302621/

Yeah, no. [I support it](https://github.com/Chizaruu/com.tsk.ide.vscode). It would be reckless for the vscode team to kill off this page.

**Comment by TeaDrinkingProgrammer** [comment]
> > Hopefully, I can prevent people from wasting time on this. https://forum.unity.com/threads/update-on-the-visual-studio-code-package.1302621/
> 
> Yeah, no. [I support it](https://github.com/Chizaruu/com.tsk.ide.vscode). It would be reckless for the vscode team to kill off this page.

Oh, ok. In that case I missed your updated version because I only went of the Unity communication. The plugin is still not officially supported though, so in this case, we could turn this issue around to point to your repo?

**Comment by gregvanl** [comment]
Closing as Microsoft just released the Unity for Visual Studio Code extension
https://marketplace.visualstudio.com/items?itemName=visualstudiotoolsforunity.vstuc
See https://devblogs.microsoft.com/visualstudio/announcing-the-unity-extension-for-visual-studio-code/ for the announcement.
People can leave feedback on the new extension and Unity in VS Code experience at https://github.com/microsoft/vscode-dotnettools/issues

**Pull request closed without merging**